submit-api/src/submit_api/models/queries/package.py

```diff
@@ -132,7 +133,21 @@ def update_package_status(package_id, session, package=None):
 
     @classmethod
     def get_account_project_packages(cls, account_id: int):
```

Comment:
what will happen to the components that were using this method with the old query

Reply:
I checked, and this method is only used in the New User Form and Update Role Form. The corresponding hook is useGetAccountPackagesByAccountId.

Reply:
okay very well, could we plz change the name of this method to get_account_project_unique_packages

Reply:
or %latest_packages